main.go

```diff
@@ -167,3 +174,35 @@ func client(args []string) {
 		log.Fatal(err)
 	}
 }
+
+// validate the provided Remotes configuration is valid
+func validateRemotes(remotes []string) error {
+	uniqueRemotes := []string{}
+
+	for _, newRemote := range remotes {
```

Comment:
can't we create a list/slice with ports and use the "unique" package and check the size ?

Reply:
For future proof scenarios, in case we want to extend the validations, having it behind a method is easier.